### PR TITLE
[DOCU-2050] Gateway 2.7 changelog

### PR DESCRIPTION
**Pull request by lena-larionova**
### Summary

Changelog for Gateway 2.7 (OSS and EE).

### Reason
Release.

### Testing
netlify preview: https://deploy-preview-3488--kongdocs.netlify.app/gateway/changelog/

### CODE REVIEW

**Comment by lena-larionova** [comment]
@PaulFischer-Kong this is the draft for the 2.7 changelog. There are still a few pieces missing, mostly around OIDC and some links, but otherwise this is ready for you to look at.

File: app/gateway/changelog.md [hunk 1]
```diff
@@ -5,12 +5,301 @@ no_version: true
...
+  custom path for the cache.
+* The `/license/report` API endpoint now provides
+[monthly throughput usage reports](/gateway/2.7.x/plan-and-deploy/licenses/report).
+
+#### Dev Portal
+* The Dev Portal API now supports `sort_by={attribute}` and `sort_desc`
+query parameters for sorted list results.
+* At Dev Portal account and admin account creation, you can now use the `email` of
+the `consumer_claim` to either match with an existing consumer or create a
+new account.
```
Comment (falondarville) [inline app/gateway/changelog.md:35]:
```suggestion
* For Dev Portals with the [OpenID Connect](/hub/kong-inc/openid-connect) plugin enabled, login is successful on the first try if the credentials provided are valid and stored in IDP. If the email does not match existing data associated with the account, a new account is automatically created.
```

The current text requires knowledge of the codebase. I updated to what the user can expect from this change.

File: app/gateway/changelog.md [hunk 2]
```diff
@@ -5,12 +5,301 @@ no_version: true
...
+* Added TLSv1.3 support for the Dev Portal API and GUI.
+
+#### Kong Manager
+* When using OpenID Connect to secure Kong Manager, admins are created on first
+login and roles are assigned based on their group membership.
```
Comment (Bradamant3) [inline app/gateway/changelog.md:40]:
```suggestion
* When using OpenID Connect to secure Kong Manager, you no longer need to create admins manually in Kong Manager and map their roles to your identity provider. Instead, admins are created on first
login and their roles are assigned based on their group membership in your IdP. This feature also partly resolves a problem with creating admins for both Kong Manager and Dev Portal.
```

**Review by falondarville — APPROVED** [review]

File: app/gateway/changelog.md [hunk 3]
```diff
@@ -5,12 +5,301 @@ no_version: true
...
+#### Enterprise
+* Kong Gateway now supports installations on [Debian 10 and 11](/gateway/2.7.x/install-and-run/debian/).
+* This release introduces consumer groups, a new entity that lets you
+manage custom rate limiting configuration for any defined subsets of consumers.
+To use consumer groups for rate limiting, configure the [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced)
+plugin with the `enforce_consumer_groups` and `consumer_groups` parameters,
+and use the `/consumer_groups` endpoint to manage the groups.
+  * [Consumer groups reference](/gateway/2.7.x/admin-api/consumer-groups/reference)
+  * [Consumer groups examples](/gateway/2.7.x/admin-api/consumer-groups/examples)
+* The data plane configuration cache can now be encrypted or disabled. Two new
```
Comment (mikefero) [inline app/gateway/changelog.md:21]: WDYT about either of these; `disabled` sounds weird to me in this sentence?
```suggestion
* The data plane configuration cache can now be encrypted. Two new
```

or

```suggestion
* The data plane configuration cache can now be encrypted or unencrypted. Two new
```

File: app/gateway/changelog.md [hunk 4]
```diff
@@ -5,12 +5,301 @@ no_version: true
...
+
+* [Mocking](/hub/kong-inc/mocking) (`mocking`)
+  * Added the `random_examples` parameter. Use this setting to randomly select
+  one example from a set of mocked responses.
+
+* [IP-Restriction](/hub/kong-inc/ip-restriction) (`ip-restriction`)
+  * Response status and message can now be customized
+  through configurations `status` and `message`.
+  [#7728](https://github.com/Kong/kong/pull/7728)
+    Thanks [timmkelley](https://github.com/timmkelley) for the patch!
```
Comment (mikefero) [inline app/gateway/changelog.md:108]: nit: consistency (don't think it effects the generated output doc though)
```suggestion

    Thanks [timmkelley](https://github.com/timmkelley) for the patch!
```

File: app/gateway/changelog.md [hunk 5]
```diff
@@ -5,12 +5,301 @@ no_version: true
...
+
+#### Enterprise
+* Fixed an issue with Vitals report generation. If running Vitals with InfluxDB
+and attempting to generate a report containing any status codes outside of
+2XX, 4XX, or 5XX, report generation would fail. With this fix, proxied traffic
+outside of the expected codes will not cause errors, and instead appear as
+count totals in Vitals reports.
+
+* Fixed a latency issue in hybrid mode. Previously, applying a large number of
+configuration changes to a Data Plane simultaneously caused high latency in all
```
Comment (mikefero) [inline app/gateway/changelog.md:165]: nit: consistency; other uses of `Data Plane` in this commit are all lower case

```suggestion
configuration changes to a data plane simultaneously caused high latency in all
```

**Review by mikefero — COMMENTED** [review]
Looks great just a couple of comments/suggestions

File: app/gateway/changelog.md (hunk 3 above)
Comment (mikefero) [reply app/gateway/changelog.md:21]: Ignore this; I forgot about the `off` feature with new encryption feature,

**Review by PaulFischer-Kong — APPROVED** [review]

**Pull request merged**